code/modules/client/preferences/sounds.dm

```diff
@@ -38,10 +38,20 @@
 	savefile_key = "sound_instruments"
 	savefile_identifier = PREFERENCE_PLAYER
 
-/datum/preference/numeric/volume/sound_instruments/apply_to_client_updated(client/client, value)
+/// Controls jukebox track volume
+/datum/preference/numeric/volume/sound_jukebox
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_key = "sound_jukebox"
+	savefile_identifier = PREFERENCE_PLAYER
+
+/datum/preference/numeric/volume/sound_jukebox/apply_to_client_updated(client/client, value)
 	if (!value)
 		client.mob.stop_sound_channel(CHANNEL_JUKEBOX)
```

Comment:
I think this is deprecated with the changes

Reply:
Removed in favour of adding the mute status as mentioned above.